### ISSUE TITLE
GridToolbar disappearing based on ScrollMode

### ISSUE THREAD

**Issue by PaitoAnderson** [telerik/kendo-react#9]
## I'm submitting a...
Bug report 

## Current behavior
When using a `<Grid>` component and setting the ScrollMode to `none`, the `<GridToolbar>` component doesn't render.

## Expected behavior
The `<GridToolbar>` component should render.

## Minimal reproduction of the problem with instructions
https://plnkr.co/edit/oSFk7bH9HRkbQkxWnFAe?p=preview

Test button should appear.

## Environment

Package versions:
```
+-- @progress/kendo-data-query@1.2.0
+-- @progress/kendo-react-buttons@0.3.0
+-- @progress/kendo-react-dateinputs@0.3.0
+-- @progress/kendo-react-dropdowns@0.3.0
+-- @progress/kendo-react-grid@0.3.0
+-- @progress/kendo-react-inputs@0.3.0
+-- @progress/kendo-react-intl@0.3.0
+-- @progress/kendo-react-popup@0.3.0
+-- @types/jest@22.2.0
+-- @types/lodash@4.14.104
+-- @types/node@9.4.7
+-- @types/prop-types@15.5.2
+-- @types/react@16.0.40
+-- @types/react-dom@16.0.4
+-- @types/react-hot-loader@3.0.6
+-- @types/react-redux@5.0.15
+-- @types/react-router-dom@4.2.4
+-- @types/react-router-redux@5.0.12
+-- @types/signalr@2.2.35
+-- @types/webpack@2.2.15
+-- @types/webpack-env@1.13.5
+-- aspnet-webpack@2.0.3
+-- aspnet-webpack-react@3.0.0
+-- awesome-typescript-loader@3.2.1
+-- bootstrap@4.0.0
+-- UNMET PEER DEPENDENCY cldr-data@^32.0.0
+-- css-loader@0.28.10
+-- event-source-polyfill@0.0.12
+-- extract-text-webpack-plugin@2.1.2
+-- file-loader@1.1.11
+-- history@4.7.2
+-- isomorphic-fetch@2.2.1
+-- jquery@3.3.1
+-- lodash@4.17.5
+-- moment@2.21.0
+-- node-noop@1.0.0
+-- popper.js@1.12.9
+-- UNMET PEER DEPENDENCY react@16.2.0
+-- react-dom@16.2.0
+-- react-hot-loader@3.1.3
+-- react-redux@5.0.7
+-- react-router-dom@4.2.2
+-- react-router-redux@5.0.0-alpha.9
+-- react-timeago@3.4.3
+-- redux@3.7.2
+-- redux-thunk@2.2.0
+-- signalr@2.2.2
+-- style-loader@0.20.3
+-- typescript@2.7.2
+-- url-loader@0.5.9
+-- webpack@2.5.1
+-- webpack-hot-middleware@2.21.2
`-- webpack-merge@4.1.2

npm ERR! peer dep missing: react@^15.0.0, required by react-timeago@3.4.3
npm ERR! peer dep missing: cldr-data@^32.0.0, required by @progress/kendo-react-intl@0.3.0
npm ERR! peer dep missing: ajv@^6.0.0, required by ajv-keywords@3.1.0
npm ERR! peer dep missing: ajv@^6.0.0, required by ajv-keywords@3.1.0
npm ERR! missing: hawk@3.1.3, required by node-pre-gyp@0.6.39
npm ERR! missing: mkdirp@0.5.1, required by node-pre-gyp@0.6.39
npm ERR! missing: rimraf@2.6.1, required by node-pre-gyp@0.6.39
npm ERR! missing: tar@2.2.1, required by node-pre-gyp@0.6.39
npm ERR! missing: boom@2.10.1, required by hawk@3.1.3
npm ERR! missing: cryptiles@2.0.5, required by hawk@3.1.3
npm ERR! missing: hoek@2.16.3, required by hawk@3.1.3
npm ERR! missing: sntp@1.0.9, required by hawk@3.1.3
npm ERR! missing: hoek@2.16.3, required by boom@2.10.1
npm ERR! missing: boom@2.10.1, required by cryptiles@2.0.5
npm ERR! missing: hoek@2.16.3, required by sntp@1.0.9
npm ERR! missing: minimist@0.0.8, required by mkdirp@0.5.1
npm ERR! missing: console-control-strings@1.1.0, required by npmlog@4.1.0
npm ERR! missing: readable-stream@2.2.9, required by are-we-there-yet@1.1.4
npm ERR! missing: console-control-strings@1.1.0, required by gauge@2.7.4
npm ERR! missing: string-width@1.0.2, required by gauge@2.7.4
npm ERR! missing: strip-ansi@3.0.1, required by gauge@2.7.4
npm ERR! missing: code-point-at@1.1.0, required by string-width@1.0.2
npm ERR! missing: is-fullwidth-code-point@1.0.0, required by string-width@1.0.2
npm ERR! missing: strip-ansi@3.0.1, required by string-width@1.0.2
npm ERR! missing: number-is-nan@1.0.1, required by is-fullwidth-code-point@1.0.0
npm ERR! missing: ansi-regex@2.1.1, required by strip-ansi@3.0.1
npm ERR! missing: string-width@1.0.2, required by wide-align@1.1.2
npm ERR! missing: combined-stream@1.0.5, required by request@2.81.0
npm ERR! missing: hawk@3.1.3, required by request@2.81.0
npm ERR! missing: mime-types@2.1.15, required by request@2.81.0
npm ERR! missing: safe-buffer@5.0.1, required by request@2.81.0
npm ERR! missing: delayed-stream@1.0.0, required by combined-stream@1.0.5
npm ERR! missing: combined-stream@1.0.5, required by form-data@2.1.4
npm ERR! missing: mime-types@2.1.15, required by form-data@2.1.4
npm ERR! missing: extsprintf@1.0.2, required by jsprim@1.4.0
npm ERR! missing: extsprintf@1.0.2, required by verror@1.3.6
npm ERR! missing: mime-db@1.27.0, required by mime-types@2.1.15
npm ERR! missing: safe-buffer@5.0.1, required by tunnel-agent@0.6.0
npm ERR! missing: glob@7.1.2, required by rimraf@2.6.1
npm ERR! missing: fs.realpath@1.0.0, required by glob@7.1.2
npm ERR! missing: inflight@1.0.6, required by glob@7.1.2
npm ERR! missing: inherits@2.0.3, required by glob@7.1.2
npm ERR! missing: minimatch@3.0.4, required by glob@7.1.2
npm ERR! missing: once@1.4.0, required by glob@7.1.2
npm ERR! missing: path-is-absolute@1.0.1, required by glob@7.1.2
npm ERR! missing: once@1.4.0, required by inflight@1.0.6
npm ERR! missing: wrappy@1.0.2, required by inflight@1.0.6
npm ERR! missing: brace-expansion@1.1.7, required by minimatch@3.0.4
npm ERR! missing: balanced-match@0.4.2, required by brace-expansion@1.1.7
npm ERR! missing: concat-map@0.0.1, required by brace-expansion@1.1.7
npm ERR! missing: block-stream@0.0.9, required by tar@2.2.1
npm ERR! missing: fstream@1.0.11, required by tar@2.2.1
npm ERR! missing: inherits@2.0.3, required by tar@2.2.1
npm ERR! missing: inherits@2.0.3, required by block-stream@0.0.9
npm ERR! missing: graceful-fs@4.1.11, required by fstream@1.0.11
npm ERR! missing: inherits@2.0.3, required by fstream@1.0.11
npm ERR! missing: mkdirp@0.5.1, required by fstream@1.0.11
npm ERR! missing: rimraf@2.6.1, required by fstream@1.0.11
npm ERR! missing: fstream@1.0.11, required by tar-pack@3.4.0
npm ERR! missing: once@1.4.0, required by tar-pack@3.4.0
npm ERR! missing: readable-stream@2.2.9, required by tar-pack@3.4.0
npm ERR! missing: rimraf@2.6.1, required by tar-pack@3.4.0
npm ERR! missing: tar@2.2.1, required by tar-pack@3.4.0
npm ERR! missing: fstream@1.0.11, required by fstream-ignore@1.0.5
npm ERR! missing: inherits@2.0.3, required by fstream-ignore@1.0.5
npm ERR! missing: minimatch@3.0.4, required by fstream-ignore@1.0.5
npm ERR! missing: wrappy@1.0.2, required by once@1.4.0
npm ERR! missing: buffer-shims@1.0.0, required by readable-stream@2.2.9
npm ERR! missing: core-util-is@1.0.2, required by readable-stream@2.2.9
npm ERR! missing: inherits@2.0.3, required by readable-stream@2.2.9
npm ERR! missing: isarray@1.0.0, required by readable-stream@2.2.9
npm ERR! missing: process-nextick-args@1.0.7, required by readable-stream@2.2.9
npm ERR! missing: string_decoder@1.0.1, required by readable-stream@2.2.9
npm ERR! missing: util-deprecate@1.0.2, required by readable-stream@2.2.9
npm ERR! missing: safe-buffer@5.0.1, required by string_decoder@1.0.1
```

Browser:
- Chrome (desktop) version 65.0.3325.146

System:
- TypeScript version: 2.7.2
- Node version: v8.9.4, v5.7.1
- Platform:  Windows

**Comment by Xizario** [comment]
Fixed in `v0.5.0-dev.201803131253`, [Live example](https://stackblitz.com/edit/react-grid-toolbar).
Please refer to the [this article](https://www.telerik.com/kendo-react-ui/components/installation/development-builds/) on how to obtain the development builds.

**Comment by PaitoAnderson** [comment]
Thanks @Xizario!
